python/skupper_router_internal/dispatch.pyi

```diff
@@ -17,7 +17,13 @@
 # under the License
 #
 
-"""Type stubs for objects implemented in the C extension module"""
+"""Type stubs for objects implemented in the C extension module.
```

Comment:
Perhaps mention here that the definition for IoAdaptor and LogAdaptor can be found in python_embedded.c ?

Reply:
I wrote that in `dispatch.py` already; too bad there does not seem to be one obvious place for the information, due to the nature of .pyi.

Analogy with C would suggest to put info into header file, that is pyi, but in my experience mostly documentation comment in python is in py files.